src/component/AllCategories/CategoryBlock.css

```
@@ -0,0 +1,27 @@
+
+.personal-item {
```

personal-item не совсем корректно, лучше называть category-item, personal лучше не использовать в этом документе

```suggestion
.personal-item {
```